docs/pages/management/guides/joining-nodes-kubernetes-serviceaccount.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Joining Nodes via Kubernetes ServiceAccount token
+description: Use Kubernetes ServiceAccount tokens to join Nodes running in the same Kubernetes cluster than the Auth Service.
+---
+
+This guide will explain how to use the **Kubernetes join method** to configure
+Teleport instances to join your Teleport cluster without sharing any
+secrets when they are running in the same Kubernetes cluster than the Auth
+servers.
+
+<ScopedBlock scope="cloud">
+
+The Kubernetes join method is not available in Teleport Cloud as it requires the
+joining node to run in the same Kubernetes cluster as the Auth nodes.
+
+</ScopedBlock>
+
+The Kubernetes join method is available in self-hosted versions of Teleport 12+.
+It is available to any Teleport Node or Proxy running in the same Kubernetes
+cluster than the Auth server.
+
+The Kubernetes ClusterRole `system:auth-delegator` is required on the Auth
+servers ServiceAccount. Clusters deployed with the [`teleport-cluster` Helm
+chart](../../reference/helm-reference/teleport-cluster.mdx) version 12 or
+higher have the correct roles by default.
+
+## Prerequisites
+
+- A running Teleport cluster in Kubernetes. For details on how to set this up,
+  see [Guides for running Teleport using Helm](../../deploy-a-cluster/helm-deployments.mdx).
+- Editor access to the Kubernetes cluster running the Teleport cluster.
+  You must be able to create Namespaces and Deployments.
+- A Teleport user with `access` role, or any other role allowing to connect to
+  applications wearing the labels `app: demo-app`
+- Either the Teleport `editor` role, or the ability to `kubectl exec` into the
+  existing Teleport Auth Nodes.
+
+## Step 1/5. Create the Kubernetes provision token
```

Comment:
```suggestion
## Step 1/5. Create the Kubernetes join token
```
Should we be consistent with other docs guides here, or is this different enough that we should use another term?